clustintime/clustintime.py

```diff
@@ -124,17 +125,17 @@ def clustintime(
     nscans = corr_map.shape[0]
     indexes = range(corr_map.shape[0])
 
-    if processing != None:
+    if processing is not None:
```

Comment:
Can `processing` be falsy but not `None` here? As in an empty string `""`, an empty list, `[]`, or a null int `0`?
If this is not the case, consider this change.
```suggestion
    if processing:
```

Reply:
Also, checking if something is None needs to use this syntax. 
`a == None` checks if a is truthy and None is truthy, and compares that (None is always falsy, so if a is `0`, `[]` or `""`, this check will pass, because both values are falsy).

If you want to make sure the variable is exactly `None`, we need to check its identity with `is`.
Consider this [SO thread](https://stackoverflow.com/questions/14247373/python-none-comparison-should-i-use-is-or).